src/registry/export_sdl.rs

```diff
@@ -131,10 +131,7 @@ impl Registry {
                 write!(sdl, "type {} ", name).ok();
                 if let Some(implements) = self.implements.get(name) {
                     if !implements.is_empty() {
-                        write!(sdl, "implements ").ok();
-                        for interface in implements {
-                            write!(sdl, "& {} ", interface).ok();
-                        }
+                        write!(sdl, "implements {} ", implements.iter().map(AsRef::as_ref).collect::<Vec<&str>>().join(" & ")).ok();
```

Comment:
By the way, the previous version [adhered to the spec fine](https://spec.graphql.org/June2018/#ImplementsInterfaces). Although this version does look prettier.

Reply:
Oh...funny that they have it different in two places.